packages/module/src/ConsoleInternal/components/catalog/_catalog.scss

```diff
@@ -7,7 +7,7 @@ $pfext-m-catalog-tile-width: 260px;
 
 $catalog-capability-level-icon-left: -20px;
 $catalog-capability-level-icon-top: 4px;
-$catalog-capability-level-inactive-color: var(--pf-v5-global--Color--400);
+$catalog-capability-level-inactive-color: var(--pf-v6-global--Color--400);
```

Comment:
it's worth noting that the vast majority of CSS vars that start with `--pf-v6-global` will all need to be updated since that pattern has changed to `--pf-t--global` except for a few special cases. 

I'm not sure where this color is used, but we have `--pf-t--global--text--color--disabled` for disabled things, and `--pf-t--global--text--color--on-disabled` for text used on a disabled background (like disabled button text). We also have `--pf-t--global--text--color--subtle` which is the "secondary" (grey) text color, often for stuff like subtitles/descriptions.